Pod/Classes/Form/PDFFormViewController.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-open class PDFFormViewController: NSObject {
+open class PDFFormViewController {
```

Comment:
This library supports Objective-C? Theres no mention of Objective-C support anywhere in the README or an Objective-C version of the example project.